docs/node-operators/delegation-tiebreak.mdx

```diff
@@ -1,71 +1,75 @@
 
-2.&nbsp; This list of tied keys is sorted alphanumerically using the [canonical ascii representation](http://support.ecisolutions.com/doc-ddms/help/reportsmenu/ascii_sort_order_chart.htm)
+1. This list of tied keys is sorted alphanumerically using the [canonical ASCII representation](http://support.ecisolutions.com/doc-ddms/help/reportsmenu/ascii_sort_order_chart.htm).
 
-3.&nbsp; The sorted list is randomized using [Random.org’s List Randomizer](https://www.random.org/lists/?mode=advanced);
+1. The sorted list is randomized using the Random.org advanced [List Randomizer](https://www.random.org/lists/?mode=advanced).
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp; The [VRF](../glossary#vrf) seed from the last epoch of the previous cycle is used as the “persistent identifier”
+  The [VRF](/glossary#verifiable-random-function-vrf) seed from the last epoch of the previous cycle is used as the “persistent identifier”.
 
 ## Verifying the Delegation Program Random Tiebreak
 
 ### Requirements
 
-If you would like to verify the Delegation Program random tiebreak result you will need the following:
-- [Verifiable Random Function (VRF)](../glossary/#vrf) seed from the last epoch of the previous cycle
-- List of tied wallet addresses
-- Sorting tool (eg, [https://onlinestringtools.com/sort-strings](https://onlinestringtools.com/sort-strings))
-- Randomizing tool ([https://www.random.org/lists/?mode=advanced](https://www.random.org/lists/?mode=advanced))
+To verify the Delegation Program random tiebreak result, you must have:
 
-### Instructions
-
-#### **Step 1:** Identify the VRF seed from the last epoch of the previous cycle
-
-a. &nbsp; The VRF seed will be posted with the list of the Top 240 eligible delegatees
-
-b. &nbsp; If you like you can obtain the VRF seed yourself by:
+- The [Verifiable Random Function [VRF](/glossary#verifiable-random-function-vrf) seed from the last epoch of the previous cycle.
+- The list of tied wallet addresses.
+- A sorting tool. For example, [https://onlinestringtools.com/sort-strings](https://onlinestringtools.com/sort-strings).
+- A randomizing tool. For example, [https://www.random.org/lists/?mode=advanced](https://www.random.org/lists/?mode=advanced).
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; i.&nbsp; Query [https://graphql.minaexplorer.com/](https://graphql.minaexplorer.com), using last epoch of the previous cycle,  with the following:
+### Instructions
 
-```
-query epochSeed {
-  blocks(limit: 1, query: {canonical: true, protocolState: {consensusState: {epoch: 33}}}) {
-    protocolState {
-      consensusState {
-        stakingEpochData {
-          seed
+1. Identify the VRF seed from the last epoch of the previous cycle.
+
+    - The VRF seed is posted with the list of the Top 240 eligible delegatees.
+    - To obtain the VRF seed by yourself, you can query the [https://graphql.minaexplorer.com/](https://graphql.minaexplorer.com) using the last epoch of the previous cycle with:
```

Comment:
In general, it's a bit confusing to keep on referring to the seed you need as "the seed from the last epoch of the previous cycle". This is a partially inaccurate description of an Ouroboros implementation detail that users need not be aware of.

This concept is called the "epoch seed", and the specific instance we want to query for a given epoch is the "active staking epoch seed". The query below is querying exactly this -- it looks at the latest canonical block in the chain at the specified epoch you are querying, and it returns the staking epoch seed for that block. The fact that this epoch seed was determined at some point in a prior epoch isn't an important detail for users to be aware of.

Reply:
I'll move this feedback to a new PR so we can get this one merged, love this teaching, thank you @nholland94

Reply:
Saved this follow up task to GitHub issue https://github.com/o1-labs/docs2/issues/612